packages/date/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/date",
-	"version": "2.0.0",
+	"version": "2.1.0",
```

Comment:
This needs to be removed. It should be handled by Lerna when publishing the package to npm. This case doesn't depend on other internal packages from our monorepo so technically it might work. However if we would have any of those it would be simply unusable when installed from npm.

Reply:
Ahhhh okay, sorry, I'm a lerna newbie 😞

Reply:
No worries, it will bump to 2.1.1 otherwise :)